supplementary_style_guide/style_guidelines/links.adoc

```diff
@@ -20,11 +20,19 @@ For more information about <topic>, see xref:<link>[<link_text>].
 
 Follow these guidelines when linking externally:
 
-* Always include link text. Bare URLs (URLs without explanatory information) are unhelpful because they do not provide adequate context about the link target.
+* Always include accurate link text. Bare URLs (URLs without explanatory information) are unhelpful because they do not provide adequate context about the link target.
```

Comment:
"Always include link text, and ensure that the link text is descriptive. Bare URLs (URLs without explanatory information) and generic link text (such as "click here") are unhelpful because they do not provide adequate context about the link target."

Reply:
I think your point about avoiding generic link text is good and should be included too, preferably in the second sentence.